thorlcr/graph/thgraphmaster.ipp

```diff
@@ -197,7 +201,17 @@ public:
         CriticalBlock b(wuDirty);
         dirty = true;
     }
-    
+// Track spills
+    CriticalSection critSpillTrack;
```

Comment:
Are these updated from multiple threads?  If so using atomic (RelaxedAtomic) is probably cleaner, although not necessarily more efficient, than a critical section - especially since they are accessed without a critical section.  (RelaxedAtomic::store_max).